ui/page/utils.go

```diff
@@ -119,3 +121,41 @@ func handleSubmitEvent(editors ...*widget.Editor) bool {
 	}
 	return submit
 }
+
+func SwitchEditors(keyEvent chan *key.Event, editors ...*widget.Editor) {
+	for i := 0; i < len(editors); i++ {
+		if editors[i].Focused() {
+			if components.HandleTabEvent(keyEvent) {
+				if i == len(editors)-1 {
+					editors[0].Focus()
+				} else {
+					editors[i+1].Focus()
+				}
+			}
+		}
+	}
+}
+
+func SwitchRestoreEditors(editors ...*widget.Editor) {
+	for i := 0; i < len(editors); i++ {
+		if editors[i].Focused() {
+			if i == len(editors)-1 {
+				editors[0].Focus()
+			} else {
+				editors[i+1].Focus()
+			}
+		}
+	}
+}
+
+func SwitchSeedEditors(editors []decredmaterial.RestoreEditor) {
+	for i := 0; i < len(editors); i++ {
+		if editors[i].Edit.Editor.Focused() {
+			if i == len(editors)-1 {
+				editors[0].Edit.Editor.Focus()
+			} else {
+				editors[i+1].Edit.Editor.Focus()
+			}
+		}
+	}
+}
```

Comment:
Since this is only used once, no need exporting it and it can be moved to the create restore page.
```suggestion
func switchSeedEditors(editors []decredmaterial.RestoreEditor) {
	for i := 0; i < len(editors); i++ {
		if editors[i].Edit.Editor.Focused() {
			if i == len(editors)-1 {
				editors[0].Edit.Editor.Focus()
			} else {
				editors[i+1].Edit.Editor.Focus()
			}
		}
	}
}
```